recipes/shield/all/conanfile.py

```diff
@@ -0,0 +1,32 @@
+from conans import ConanFile, tools, CMake
+import os
+
+class ShieldConan(ConanFile):
+    name = "shield"
+    version = "0.1"
```

Comment:
```suggestion
```

do not add version, so you can re-use the same recipe to any version

Reply:
done